crates/matrix-sdk-ui/src/unable_to_decrypt_hook.rs

```diff
@@ -241,13 +246,17 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_deduplicates_utds() {
+    async fn build_test_client() -> Client {
+        test_client_builder(None).build().await.unwrap()
+    }
```

Comment:
Instead of this new helper, can you reuse either `no_retry_test_client` or `logged_in_client` from the test helper suite?

Reply:
Sure, if you like. I didn't need the "no retry" bit, so thought I might as well just use the unadulterated client. But it doesn't matter.